src/ovms.h

```diff
@@ -79,6 +79,14 @@ typedef enum OVMS_BufferType_enum {
     OVMS_BUFFERTYPE_HDDL,
 } OVMS_BufferType;
 
+typedef enum OVMS_ServableState_enum {
```

Comment:
In TFS and KFS the endpoint name is called GetModelStatus or ModelStatus - why do we call it differently in C-API (State)?